{{cookiecutter.project_slug}}/requirements.txt

```diff
@@ -1 +1,2 @@
 tqdm==4.32.1
+google-cloud-storage==1.23.0
```

Comment:
will move to the base image: see issue https://github.com/neuromation/template-base-image/issues/56

Reply:
let's add `neuro exec` right into `entrypoint.sh`